Mathlib/Data/Nat/Choose/Bounds.lean

```diff
@@ -50,4 +50,19 @@ theorem pow_le_choose (r n : ℕ) : ((n + 1 - r : ℕ) ^ r : α) / r ! ≤ n.cho
     exact n.pow_sub_le_descFactorial r
   exact mod_cast r.factorial_pos
 
+theorem choose_succ_le_two_pow {n k : ℕ} : (n + 1).choose k <= 2 ^ n := by
```

Comment:
Question: it seems like n and k are explicit in the rest of the file. Should they be here as well?

Reply:
You're right, I made them explicit. They just happened not to be because in my application they could be inferred!